course/src/main/java/org/openedx/course/presentation/dates/CourseDatesScreen.kt

```diff
@@ -261,19 +268,21 @@ private fun CourseDatesUI(
                                     }
                                 }
 
-                                if (courseBanner.isBannerAvailableForUserType(isSelfPaced)) {
+                                if (courseBanner.isBannerAvailableForUserType(isSelfPaced) && canShowPLSBanner) {
```

Comment:
The 'canShowPLSBanner` also needs to be considered in the Analytics call in. LaunchedEffect.